book/src/products/cross-chain-virtual-machine.md

```diff
@@ -1,31 +1,40 @@
-# Cross-Chain Virtual Machine
-*The Composable XCVM allows for cross-ecosystem communication.*
+# **[Composable’s Cross-Chain Virtual Machine](https://dali.devnets.composablefinance.ninja/products/cross-chain-virtual-machine.html#cross-chain-virtual-machine)**
```

Comment:
why did you make this a link?

Reply:
This should be addressed, but marketing has been linking titles to their deployed page in the Google doc

Reply:
You can can link to titles without making them links in markdown, it's a standard HTML feature. Example: https://dali.devnets.composablefinance.ninja/ecosystem/build-on-composable-ecosystem-development.html#the-applicants-we-are-seeking

Reply:
I kept it in based on the edits doc linked in clickup.

Reply:
Alright, @Martin-Kiss could you please remove them? :-)

Reply:
Will do. 👍